service/src/main/resources/templates/ehr_allergy_structure_template.mustache

```diff
@@ -28,6 +28,7 @@
                             <text>{{pertinentInformation}}</text>
                         </pertinentAnnotation>
                     </pertinentInformation>
+                    {{{code}}}
```

Comment:
Should there also be a code for the ehrComposition level?

Reply:
Confirmed with @dmcavenue1 that code does not need to be added at ehrComposition level for allergy structure.